src/Evolution/Systems/GeneralizedHarmonic/Constraints.hpp

```diff
@@ -551,6 +551,44 @@ void constraint_energy(
     double four_index_constraint_multiplier = 1.0);
 /// @}
 
+/// @{
+/*!
+ * \brief Computes the generalized-harmonic normalized constraint energy.
+ *
+ * \details Computes the generalized-harmonic normalized constraint energy.
+ * [Eq. (70) of \cite Lindblom2005qh with \f$m^{ab}=\delta^{ab}\f$,
+ * \f{eqnarray}{
+ * ||E||=\gamma^{ij}\delta^{ab}\delta^{cd}
+ * (\Lambda^{2}\partial_{i}g_{ac}\partial_{j}g_{bd}
+ * + \partial_{i}\Pi_{ac}\partial_{j}\Pi_{bd} +
+ * \gamma^{kl}\partial_{i}\Phi_{kac}\partial_{j}\Phi_{lbd})\gamma^{1/2}
```

Comment:
This equation looks like it compute the integrand of equation 70 of the reference above, not the integral itself. If that's what was intended, add to the docs that this is the integrand, not the integral.